Sources/Services/ImageService.swift

```diff
@@ -1,6 +1,24 @@
 class ImageService {
-    static func loadImage(mxcURL: URL, completionHandler: @escaping (Result<Image>) -> Void) {
-        MatrixAPI.default.downloadImage(mxcURL: mxcURL, completionHandler: completionHandler)
+    static func loadImage(mxcURL: URL, cryptoInfo: Attachment.Info.CryptoInfo? = nil, completionHandler: @escaping (Result<Image>) -> Void) {
```

Comment:
```suggestion
     private static func loadImage(mxcURL: URL, cryptoInfo: Attachment.Info.CryptoInfo? = nil, completionHandler: @escaping (Result<Image>) -> Void) {
```

can this be private?